package.json

```
@@ -44,32 +44,35 @@
     "license": "python3 add_license.py"
   },
   "devDependencies": {
-    "@rollup/plugin-wasm": "6.1.2",
-    "@types/node": "18.15.11",
-    "@vitest/ui": "0.30.1",
-    "happy-dom": "9.1.0",
-    "prettier": "2.8.7",
-    "prettier-plugin-organize-imports": "3.2.2",
-    "ts-node": "10.9.1",
-    "typescript": "5.0.2",
-    "vite": "4.2.1",
-    "vitest": "0.29.8"
+    "@rollup/plugin-wasm": "6.2.2",
+    "@types/node": "22.13.10",
+    "@vitest/ui": "3.0.9",
+    "happy-dom": "17.4.4",
+    "prettier": "3.5.3",
+    "prettier-plugin-organize-imports": "4.1.0",
+    "ts-node": "10.9.2",
+    "typescript": "5.8.2",
+    "vite": "6.2.2",
+    "vitest": "3.0.9"
   },
   "dependencies": {
-    "@noble/ed25519": "2.0.0",
-    "@noble/hashes": "1.3.0",
-    "@types/secp256k1": "4.0.3",
-    "@types/secure-random": "1.1.0",
+    "@noble/ed25519": "2.2.3",
+    "@noble/hashes": "1.7.1",
+    "@types/secp256k1": "4.0.6",
+    "@types/secure-random": "1.1.3",
     "blakejs": "1.2.1",
-    "change-case": "4.1.2",
-    "decimal.js": "10.4.3",
-    "reflect-metadata": "0.1.13",
-    "secp256k1": "5.0.0"
+    "change-case": "5.4.4",
+    "decimal.js": "10.5.0",
+    "reflect-metadata": "0.2.2",
+    "secp256k1": "^5.0.1"
   },
   "publishConfig": {
     "registry": "https://registry.npmjs.org"
   },
   "overrides": {
     "elliptic": "6.5.7"
```

```suggestion
    "elliptic": "^6.6.1"
```

While updating dependencies, it'd be good to also update or undo https://github.com/radixdlt/typescript-radix-engine-toolkit/commit/852a6adeee55f7d74d759da935e1faa2cb3bc8a2 instead of continuing to force an outdated version with known vulnerabilities.